core/CPUTopology.cpp

```diff
@@ -202,12 +210,28 @@ olympia::CoreTopologySimple::CoreTopologySimple(){
             "cpu.core*.lsu.ports.in_cache_free_req"
         },
         {
-            "cpu.core*.dcache.ports.out_biu_req",
+            "cpu.core*.dcache.ports.out_l2cache_req",
```

Comment:
Can we add a `L2Cache.md` file documenting modeled microarchitcture of L2Cache? Such a document would help in understanding the code, and would enable new users and higher adoption by the community. 

For example, it can include:
* a description of various ports of L2Cache and their intended purpose
* various structures and pipelines used in L2cache
* stage by stage behavior of each pipeline

Reply:
Sure Arup.

Where should this L2Cache.md file reside?
I already have the info available in the presentation I prepared. Just need to port it over! :)